datasketches-memory-java9/src/main/java/module-info.java

```diff
@@ -22,5 +22,5 @@
     requires java.logging;
     requires jdk.unsupported;
 
-    exports org.apache.datasketches.memory;
+    exports org.apache.datasketches.memory.internal;
```

Comment:
I think this should stay as it was, since we do not wish to expose internals to users but rather the public facing api package instead.

Reply:
The problem is that exporting org.apache.datasketches.memory throws an error because that package under java9 is empty.  We would have to insert either a dummy class or duplicate some class from java8 just so that it is not empty.  
I changed it to -.internal to eliminate the error.   Perhaps the answer is to eliminate the export entirely.

Reply:
The approach that is in the master branch uses the `Map` implementation which is identical for the reasons that you describe.  Do you know how a JPMS module with no export behaves?  Is everything visible by default?

I think we should retain the dummy `Map` implementation to preserve the structure.  I used Map originally because there are not many LoC being duplicated.

Reply:
That sounds fine